cli/flags.rs

```diff
@@ -590,6 +608,15 @@ pub fn flags_from_vec(
       argv.extend(vec![source_file.to_string(), out_file.to_string()]);
       DenoSubcommand::Bundle
     }
+    ("completions", Some(completions_match)) => {
+      let shell: &str = completions_match.value_of("shell").unwrap();
+      create_cli_app().gen_completions_to(
+        "deno",
+        Shell::from_str(shell).unwrap(),
+        &mut std::io::stdout(),
+      );
+      std::process::exit(0);
```

Comment:
Is it necessary to exit forcefully here?

Reply:
I can add another `DenoSubcommand` and exit in `main.rs`